core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileArgs.scala

```diff
@@ -143,6 +146,11 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
     Right(Unit)
   }
 
+  // verify that either driverlog or eventlog is specified
+  validateOpt(driverlog, eventlog) {
```

Comment:
so the driver log option is weird if you pass multiple event logs or directory of event logs.   I think you want to make sure only a single event log (or directory in case of databricks) is processed when a driver log is passed.   Not sure if you can do this here, you might need to do it later in the code.

Reply:
Thanks! Updated this check in the code.